Detections/SecurityEvent/AuditPolicyManipulation_using_auditpol.yaml

```diff
@@ -0,0 +1,85 @@
+id: 66276b14-32c5-4226-88e3-080dacc31ce1
+name: Audit policy manipulation using auditpol utility
+description: |
+   This detects attempt to manipulate audit policies using auditpol command.
+   This technique was seen in relation to Solorigate attack but the results can indicate potential  malicious activity used in different attacks.
+   Refer to auditpol syntax: https://docs.microsoft.com/windows-server/administration/windows-commands/auditpol  
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceProcessEvents
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+relevantTechniques:
+  - T1204
+query: |
+  let timeframe = 1d;
+  let AccountAllowList = dynamic(['SYSTEM']);
+  let SubCategoryList = dynamic(["Logoff", "Account Lockout", "User Account Management", "Authorization Policy Change"]); // Add any Category in the list to be allowed or disallowed
+  let tokens = dynamic(["clear", "remove", "success:disable","failure:disable"]); // 
+  (union isfuzzy=true
+  (
+  SecurityEvent
+  | where TimeGenerated >= ago(timeframe)
+  //| where Process =~ "auditpol.exe" 
+  | where CommandLine has_any (tokens)
+  | where AccountType != "Machine" and Account !in (AccountAllowList)
+  | parse CommandLine with * "/subcategory:" subcategorytoken
+  | extend SubCategory = tostring(split(subcategorytoken, "\"")[1]) , Toggle =  tostring(split(subcategorytoken, "\"")[2])
+  | where SubCategory in (SubCategoryList) //use in for inclusion or !in for exclusion
+  | where Toggle !in ("/failure:disable", " /success:enable /failure:disable") // use this filter if required to exclude certain toggles
+  | project TimeGenerated, Computer, Account, SubjectDomainName,  SubjectUserName, Process, ParentProcessName,  CommandLine, SubCategory, Toggle
+  | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer
+  ),
+  (
+  DeviceProcessEvents
+  | where TimeGenerated >= ago(timeframe)
+  | where InitiatingProcessFileName =~ "auditpol.exe" 
+  | where InitiatingProcessCommandLine has_any (tokens)
+  | where AccountName !in (AccountAllowList)
+  | parse InitiatingProcessCommandLine with * "/subcategory:" subcategorytoken
+  | extend SubCategory = tostring(split(subcategorytoken, "\"")[1]) , Toggle =  tostring(split(subcategorytoken, "\"")[2])
+  | where SubCategory in (SubCategoryList) //use in for inclusion or !in for exclusion
+  | where Toggle !in ("/failure:disable", " /success:enable /failure:disable") // use this filter if required to exclude certain toggles
+  | project TimeGenerated, DeviceName, AccountName, InitiatingProcessAccountDomain, InitiatingProcessAccountName, InitiatingProcessFileName, InitiatingProcessParentFileName,  InitiatingProcessCommandLine, SubCategory, Toggle
```

Comment:
Include Type in the project so that the results are clear on what log it came from.